README.rst

```diff
@@ -17,11 +17,11 @@ handling of files and images a breeze.
    pypi. github would render relative paths correctly.
 
 +--------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------+
-| .. image:: https://raw.githubusercontent.com/divio/django-filer/master/docs/_static/filer_2.png        | .. image:: https://raw.githubusercontent.com/divio/django-filer/master/docs/_static/filer_3.png        |
+| .. image:: https://raw.githubusercontent.com/django-cms/django-filer/master/docs/_static/filer_2.png        | .. image:: https://raw.githubusercontent.com/django-cms/django-filer/master/docs/_static/filer_3.png        |
```

Comment:
The table markup is broken due to the different length of these URLs now.

It's a problem in the middle & at the end. Hopefully this screenshot illustrates what I'm talking about;
<img width="181" alt="Screenshot 2021-11-10 at 23 56 46" src="https://user-images.githubusercontent.com/1461191/141212412-10d531ae-287a-4f11-b1cf-f723b84c9117.png">
w.

Reply:
In VSCode it looks like that:

![Bildschirmfoto 2021-11-11 um 14 50 56](https://user-images.githubusercontent.com/39345336/141309428-ec77076b-4876-4d55-9d7e-096319d9ed5a.png)

Reply:
Yeah the `+` needs to lineup with the pipes (`|`) to be valid & pass the checks I think